Clicker/SectionControllers/Drafts/DraftSectionController.swift

```diff
@@ -35,8 +36,10 @@ class DraftSectionController: ListSectionController, DraftCellDelegate {
         guard let containerSize = collectionContext?.containerSize else {
             return .zero
         }
-        let textHeight = draft.text.height(withConstrainedWidth: constrainedTextWidth, font: ._18HeavyFont)
-        return CGSize(width: containerSize.width, height: textHeight + 65)
+        let calculatedTextHeight = draft.text.height(withConstrainedWidth: constrainedTextWidth, font: ._18HeavyFont)
+        let adjustedTextHeight = calculatedTextHeight + 65
```

Comment:
is there a reason for the `65`? if so can it be a constant?

Reply:
also maybe add comment explaining the 65

Reply:
I'm not actually sure about the 65 myself! It was hardcoded in the old code so I kept it because I assumed that's what the padding is calculated as. I will confirm with the design later!